.github/workflows/java_cdk.yml

```diff
@@ -9,5 +9,41 @@ on:
     paths:
       - '.github/workflows/java_cdk.yml'
       - 'airbyte-ci/java-cdk/**'
-      - 'airbyte-ci/java/**'
+      - 'airbyte-cdk/java/**'
   workflow_dispatch:
+jobs:
+  java_cdk_ci:
+    name: Java CDK CI
+    timeout-minutes: 60
+    runs-on: conn-prod-xlarge-runner
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.inputs.repo }}
+          ref: ${{ github.event.inputs.gitref }}
+      #TODO: Use python baked into the runner, currently this is on 3.8.10
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+          
+      - name: Install deps
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+      - name: Run Dagger
+        run: |
+          export _EXPERIMENTAL_DAGGER_CLOUD_TOKEN="p.eyJ1IjogIjFiZjEwMmRjLWYyZmQtNDVhNi1iNzM1LTgxNzI1NGFkZDU2ZiIsICJpZCI6ICJlNjk3YzZiYy0yMDhiLTRlMTktODBjZC0yNjIyNGI3ZDBjMDEifQ.hT6eMOYt3KZgNoVGNYI3_v4CC-s19z8uQsBkGrBhU3k"
+          export _EXPERIMENTAL_DAGGER_RUNNER_HOST="unix:///var/run/buildkit/buildkitd.sock"
+          export PREFECT_API_URL=""
+          DAGGER_CLI_COMMIT="6ed6264f1c4efbf84d310a104b57ef1bc57d57b0"
+          DAGGER_TMP_BINDIR="/tmp/dagger_${DAGGER_CLI_COMMIT}"
+          export _EXPERIMENTAL_DAGGER_CLI_BIN="$DAGGER_TMP_BINDIR/dagger"
+          if [ ! -f  "$_EXPERIMENTAL_DAGGER_CLI_BIN" ]; then
+            mkdir -p "$DAGGER_TMP_BINDIR"
+            curl "https://dl.dagger.io/dagger/main/${DAGGER_CLI_COMMIT}/dagger_${DAGGER_CLI_COMMIT}_$(uname -s | tr A-Z a-z)_$(uname -m | sed s/x86_64/amd64/).tar.gz" | tar xvz -C "$DAGGER_TMP_BINDIR"
+          fi
+          cd airbyte-ci/connectors/cdk
+          poetry install
+          poetry run aircmd plugin install cdk --local .
+          poetry run aircmd cdk java ci --scan
```

Comment:
When I run `poetry run aircmd cdk java ci --scan` locally, I get this error:

```
dagger.exceptions.QueryError: lstat /tmp/buildkit-mount2440875622/scan-journal.log: no such file or directory
```

Is this expected? Do I need to run the dagger commands in order to work properly? Locally, the `cdk java ci` command works fine, just not with `--scan` suffixed.

Reply:
Looks like the same is triggering in CI: https://github.com/airbytehq/airbyte/actions/runs/5672355854/job/15371392305?pr=28725#step:6:963

Reply:
Ah, i only tested without scan. I think this is due to the location of where we are invoking it being different than the root of the project, which is where we write scan-journal as specified in settings.grádale. I’ll push a fix for this today and submit a or to your branch